src/content/docs/en/guides/deploy/fleek.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Deploy your Astro Site to Fleek
+description: How to deploy your Astro site to the web on Fleek.
+type: deploy
+i18nReady: true
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
+
+You can use [Fleek](http://fleek.xyz/) to deploy an Astro site to their edge-optimized decentralized network.
+
+This guide gives a complete walkthrough of deploying your Astro site to Fleek using the Fleek UI and CLI.
+
+
+## Project configuration
+
+In your `astro.config.mjs`, you can configure your project to deploy to Fleek by setting the `output` to `static`:
+
+```js title="astro.config.mjs" ins={5}
+import { defineConfig } from 'astro/config';
+
+// https://astro.build/config
+export default defineConfig({
+    output:'static',
+});
+```
```

Comment:
Considering Astro's default output is set to `static` and Fleek supporting `static` websites, this section can be trimmed

Reply:
Awesome! Thank you, Atharva! You are right.